source/classes.tex

```diff
@@ -4142,8 +4142,8 @@
 the function selected by overload resolution.
 \begin{note}
 Because access control applies to the declarations named, if access control is applied to a
-typedef name, only the accessibility of the typedef name itself is considered.
-The accessibility of the entity referred to by the typedef is not considered.
+\grammarterm{typedef-name}, only the accessibility of the \grammarterm{typedef-name} itself is considered.
```

Comment:
I think it would be more in line with the normative wording to say "[...] only the accessibility of the typedef or type alias declaration itself is considered. [...]" -- as we said above, it's the declaration named, not the name, whose access is considered. (Access control applies to names, but it looks like the wording describes access(ibility) itself as a property of class members and base classes, not of names.)

Reply:
fixed